lib/ssl/src/tls_gen_connection.erl

```diff
@@ -180,37 +181,11 @@ initialize_tls_sender(#state{static_env = #static_env{
 %%====================================================================
 %% Handshake handling
 %%====================================================================
-renegotiation(Pid, WriteState) ->
-    gen_statem:call(Pid, {user_renegotiate, WriteState}).
-
-renegotiate(#state{static_env = #static_env{role = client},
-                   handshake_env = HsEnv} = State, Actions) ->
-    %% Handle same way as if server requested
-    %% the renegotiation
-    Hs0 = ssl_handshake:init_handshake_history(),
-    {next_state, connection, State#state{handshake_env = HsEnv#handshake_env{tls_handshake_history = Hs0}}, 
-     [{next_event, internal, #hello_request{}} | Actions]};
-renegotiate(#state{static_env = #static_env{role = server,
-                                            socket = Socket,
-                                            transport_cb = Transport},
-                   handshake_env = HsEnv,
-                   connection_env = #connection_env{negotiated_version = Version},
-		   connection_states = ConnectionStates0} = State0, Actions) ->
-    HelloRequest = ssl_handshake:hello_request(),
-    Frag = tls_handshake:encode_handshake(HelloRequest, Version),
-    Hs0 = ssl_handshake:init_handshake_history(),
-    {BinMsg, ConnectionStates} = 
-	tls_record:encode_handshake(Frag, Version, ConnectionStates0),
-    tls_socket:send(Transport, Socket, BinMsg),
-    State = State0#state{connection_states = 
-			     ConnectionStates,
-			 handshake_env = HsEnv#handshake_env{tls_handshake_history = Hs0}},
-    next_event(hello, no_record, State, Actions).
```

Comment:
This is called from other modules ?

Reply:
moved to tls_dtls_gen_connection

Reply:
Seems I was mistaken, will look into it more

Reply:
See comment above